lib/cdo/redshift.rb

```diff
@@ -16,6 +40,35 @@ def initialize
   end
 
   def exec(sql_query)
-    @conn.exec(sql_query)
+    result = @conn.exec(sql_query)
+    # result_status is an undocumented method that returns the integer query status.
+    # res_status is an undocumented method that returns the string identifier for a given integer query status.
+    result_status = result.res_status(result.result_status)
+    raise PostgreSQLQueryError.new(result_status) unless QUERY_SUCCESS_CODES.include? result_status
+    result
+  end
+
+  # Returns an array of table names in a Redshift schema that were loaded via DMS to stage data from Aurora.
+  def get_temporary_import_tables(schema)
```

Comment:
@bencodeorg This query returns more results (all tables) when I execute as the Redshift user `suresh`, but it does not return any of the tables that were created by Database Migration Service Replication Tasks when this query is executed by our production systems using the `dev` user.  Any thoughts on the discrepancy?

Reply:
Ah.  Looks like I needed to specify the `search_path`